docker/services/simplified_logrotate.conf

```diff
@@ -9,3 +9,15 @@
     notifempty
     dateext
 }
+
+/var/log/uwsgi/*.log {
+    missingok
+    daily
+    create 0660 simplified adm
+    rotate 13
```

Comment:
Since the uwsgi log isn't terribly large, I think I'd prefer a longer default retention time. Every once in a great while that may come in handy. Maybe 30 days? Or, perhaps the value could be "variablized" and an ENV made available for run time? Otherwise, all good.

Reply:
Apologies for merging this before you had a chance to weigh in. I'm OK with either 13 or 30 days as the retention time.

Reply:
I'm fine with 30 days. I only chose 13 so it was the same as the other logrotate configs in the container. 

If everyone is happy with 30, I can make another PR to change it.

Reply:
PR here to change the logs kept to 30: 
https://github.com/NYPL-Simplified/circulation/pull/1529